src/xenia/base/memory.h

```diff
@@ -441,6 +441,10 @@ inline void store_and_swap<std::u16string>(void* mem,
   return store_and_swap<std::u16string_view>(mem, value);
 }
 
+inline constexpr uint32_t multi_char(char a, char b, char c, char d) {
```

Comment:
`char` is signed on some platforms — if a value of over 128 was passed here, would it be treated as 0x0000008… in the formula below, or as 0xFFFFFF8…? I'm not sure how promotion is going to work here. But I'd cast them explicitly to `uint32_t` below.